turf/src/commonMain/kotlin/org/maplibre/spatialk/turf/booleans/pointInPolygon.kt

```diff
@@ -23,15 +23,15 @@ import org.maplibre.spatialk.geojson.Position
  *   Polygon
  */
 @JvmOverloads
-public fun booleanPointInPolygon(
+public fun pointInPolygon(
```

Comment:
renamed this away from the turfjs naming convention because the smurf naming feels off